Add the disable metrics page to AutoOps SM docs (#4221)
elastic/docs-content: deploy-manage/monitor/autoops/autoops-disable-metrics-collection.md

3. Delete the related lines from the file. For example,
    * to disable the collection of task-related data, delete the `tasks_management` line 
    * to disable the collection of template-related data, delete the `Templates` module 
    The following code demonstrates these examples:

    ```yaml
    receivers:
      metricbeatreceiver:
        metricbeat:
          modules:
            # Metrics
            - module: autoops_es
              hosts: ${env:AUTOOPS_ES_URL}
              period: 10s
              metricsets:
                - cat_shards
                - cluster_health
                - cluster_settings
                - license
                - node_stats
                # --- DELETE THIS LINE ---
                - tasks_management
            # --- DELETE THIS MODULE ---
            # Templates
            - module: autoops_es
              hosts: ${env:AUTOOPS_ES_URL}
              period: 24h
              metricsets:
                - cat_template
                - component_template
                - index_template
    ```
4. Save your changes to the `autoops_es.yml` file.

> 5. Restart {{agent}} so that the new settings can take effect.\

Reviewer: ```suggestion
5. Restart {{agent}} so that the new settings can take effect.
```

I assume the `\` was accidental.

2. In the `autoops_es.yml` file, locate the metric or module related to the data that you want AutoOps to stop collecting. 
3. Delete the related metricsets or module lines from the file.

    ```yaml
    receivers:
      metricbeatreceiver:
        metricbeat:
          modules:
            # Metrics
            - module: autoops_es
              hosts: ${env:AUTOOPS_ES_URL}
              period: 10s
              metricsets:
                - cat_shards
                - cluster_health
                - cluster_settings
                - license
                - node_stats
                - tasks_management <1>
            # Templates
            - module: autoops_es <2>
              hosts: ${env:AUTOOPS_ES_URL}
              period: 24h
              metricsets:
                - cat_template
                - component_template
                - index_template
    ```
    For example, 
    1. to disable the collection of task-related data, delete the `tasks_management` line
    2. to disable the collection of template-related data, delete all the lines in the `Templates` section
4. Save your changes to the `autoops_es.yml` file.

> 5. Restart {{agent}} so that the new settings can take effect.\

Reviewer 2: ```suggestion
5. Restart {{agent}} for the new settings to take effect.
```
Just a bit briefer phrasing, and also I think it looks odd having the two sentences appear without a line break in between them. I think I'd just keep it as a single paragraph, but otherwise I'd have a line break in between the sentences.
